From: Ricardo Salveti de Araujo <ricardo.salveti@canonical.com>
Subject: [PATCH] Enable module-switch-on-connect by default on Ubuntu Touch

Ugly hack, until we find a better way to store Ubuntu Touch specific changes.

Index: pulseaudio-4.0/src/daemon/default.pa.in
===================================================================
--- pulseaudio-4.0.orig/src/daemon/default.pa.in
+++ pulseaudio-4.0/src/daemon/default.pa.in
@@ -193,6 +193,12 @@ ifelse(@HAVE_X11@, 1, [dnl
 #.endif
 ])dnl
 
+### Specific changes and modules when running Ubuntu Touch
+### TODO: find a better way to handle ubuntu touch specific changes
+.ifexists /usr/bin/ubuntu-touch-session
+load-module module-switch-on-connect
+.endif
+
 ### Make some devices default
 #set-default-sink output
 #set-default-source input
